Jenkinsfile

```diff
@@ -234,6 +250,13 @@ pipeline {
                             post {
                                 always {
                                     junit testResults: "**/*/TEST*${SUREFIRE_REPORT_NAME_SUFFIX}*.xml", keepLongStdio: true
+                                    stash allowEmpty: true, name: itJacocoStashName, includes: '**/*.exec'
```

Comment:
Do we want to run test coverage for integration tests? I find this approach rather unusual. 

Normally, code coverage is recorded for unit tests primarily. Because for unit tests the code being executed is often the code being testen. 

For integration tests, there is so much code being executed during a test that the code coverage metric becomes less useful.

Reply:
Actually I agree, it's simply that I wanted to restore the previous state (where we would've been running code coverage for integration tests). If we don't want this we should disable jacoco for the integration tests. I'm happy to do this as well, it should be just a simple addition in the IT module.